SystemJSPublicPathWebpackPlugin.js

```diff
@@ -0,0 +1,69 @@
+const webpack = require("webpack");
+
+const isWebpack5 = webpack.version && webpack.version.startsWith("5.");
+
+class SystemJSPublicPathWebpackPlugin {
+  constructor(options) {
+    this.options = options || {};
+    if (!isWebpack5 && !this.options.systemjsModuleName) {
```

Comment:
to be a little more future-proof, could we do a check here for if it's `>=5` or `<5`? otherwise we'll have issues when v6 is out

Reply:
i didn't want to bring in the full `semver` library - my understanding is that version numbers don't even have to be numbers? Not sure if they have to start with numbers, either.